### DISCUSSION TITLE
JHipster .NET 3.1

### DISCUSSION THREAD

**Discussion by cvillami** [jhipster/jhipster-dotnetcore#587] [Q&A]
Hi,

In a client where I am they force me to use .NET 3.1, does anyone know if I can make use of JHipster, or how much would the adaptation cost?. Average...

If the above could not be done, does anyone know of a similar framework (Angular + .NET 3.1)? 

A greeting.

**Comment by ivanmonteiro** [comment]
You can try version 1.1.2 which I think is the last version that used .Net Core v3.1, but keep in mind that a lot of changes/bugfixes were implemented since that version.

**Comment by nicolas63** [comment]
Yes i think you can use 1.1.2 or maybe you can use actual version and downgrade to 3.1 you can do that in Directory.Build.Taget (i think you can downgrade in one day). With the second solution the generated project use ef core 5 but ef core 5 is compatible with dotnet core 3.1 (to be verify on last version)
